elastic/kibana
# [Stateful sidenav]  Remove Launch Darkly feature flag

In this PR I've removed the LaunchDarkly feature flag for the solution stateful navigation.
The feature flag was used in the `navigation` and `space` plugins.

The new navigation is available when the Space `solution` property is set to one of the 3 solution ("es", "oblt" and "security"). 

For now this will only be available **on cloud** environment (and logically not serverless). I've added a new `yml` config the Space plugin `allowSolutionVisibility` that follows the same pattern as other setting (`allowFeatureVisibility`), and I disabled it in serverless.

## How to test

* Launch Kibana
* Navigate to the Space management page (`/app/management/kibana/spaces/`)
* Edit a space
* You should not have the option to set the `solution` property.
* Edit the `kibana.dev.yml` file and add the following

```yml
xpack.cloud.id: "ftr_fake_cloud_id:aGVsbG8uY29tOjQ0MyRFUzEyM2FiYyRrYm4xMjNhYmM="
xpack.cloud.base_url: "https://cloud.elastic.co"
```
* Refresh the browser window
* You should now be able to set the `solution` view property.
